### PR TITLE
Update NuGet references

### PR DESCRIPTION
**Pull request by agocke**
References were partially updated, causing downgrading for missed projects.

/cc @dotnet/roslyn

### CODE REVIEW

File: src/ExpressionEvaluator/CSharp/Source/ResultProvider/NetFX20/project.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
     "Microsoft.NetFX20": "1.0.3",
-    "Microsoft.VisualStudio.Debugger.Engine": "14.3.25420",
-    "Microsoft.VisualStudio.Debugger.Metadata": "14.3.25420"
+    "Microsoft.VisualStudio.Debugger.Engine": "14.3.25421",
+    "Microsoft.VisualStudio.Debugger.Metadata": "14.3.25421"
```
Comment (jasonmalinowski) [inline src/ExpressionEvaluator/CSharp/Source/ResultProvider/NetFX20/project.json]: The core ResultProvider project has the references right? Why not just remove these and let it flow?

Comment (agocke) [inline src/ExpressionEvaluator/CSharp/Source/ResultProvider/NetFX20/project.json]: Makes sense. I'll do that.

**Comment by jaredpar** [comment]
:+1:

**Comment by jasonmalinowski** [comment]
:+1:

**Pull request merged**